index.js

```diff
@@ -43,6 +50,10 @@ class Main {
     this.worker = new worker.worker();
     await this.worker.init(this.exporter, metrics);
     await this.handleInitPosition();
+    if (BLOCKCHAIN === 'eth') {
```

Comment:
⚠️  Temporary: Currently testing with the Ethereum codebase. When done, would rework the rest of the code to work in the new way